pom.xml

```diff
@@ -319,14 +329,21 @@
                 <groupId>org.eclipse.che.lib</groupId>
                 <artifactId>che-websocket-terminal</artifactId>
                 <version>${che.lib.version}</version>
-                <type>zip</type>
+                <type>tar.gz</type>
                 <classifier>linux_amd64</classifier>
             </dependency>
             <dependency>
                 <groupId>org.eclipse.che.lib</groupId>
                 <artifactId>che-websocket-terminal</artifactId>
                 <version>${che.lib.version}</version>
                 <type>zip</type>
+                <classifier>linux_amd64</classifier>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.che.lib</groupId>
+                <artifactId>che-websocket-terminal</artifactId>
+                <version>${che.lib.version}</version>
+                <type>tar.gz</type>
                 <classifier>linux_arm7</classifier>
```

Comment:
for linux_amd64 you pack zip and tar.gz but for linux_arm7 only tar.gz. Can you please explain why?